SYCL/AtomicRef/device_has_aspect_atomic64_cuda_and_hip.cpp

```diff
@@ -1,3 +1,5 @@
+// FIXME unexpected pass on hip
```

Comment:
This one has been fixed on HIP in #1229. I think that the rest of HIP tests should probably be simply enabled back, i.e. we need to remove `XFAIL` from them

Reply:
PR is updated, removed not relevant XFAIL for HIP